lib/cacheWrapper.js

```diff
@@ -0,0 +1,155 @@
+/* jslint node: true */
+'use strict';
+
+var _ = require( 'lodash' );
+var Catbox = require( 'catbox' );
+var Memory = require( 'catbox-memory' );
```

Comment:
This and the var below I would think they should be `CatboxMemory` & `CatboxRedis` - Just think it makes the vars cleaner and easier to understand further on.

Unless this is some standard?